app/views/devise/registrations/edit_email.html.erb

```diff
@@ -34,14 +34,13 @@
         autocomplete: "email",
```

Comment:
Consider: using `username` over `email` for better support with password managers - detail in main comment
```suggestion
        autocomplete: "username",
```